tests/ci_build/Dockerfile.cpu_bare

```diff
@@ -8,19 +8,20 @@ ENV DEBIAN_FRONTEND noninteractive
 RUN \
     yum -y update && \
     yum install -y tar unzip wget xz git && \
-    wget http://people.centos.org/tru/devtools-2/devtools-2.repo -O /etc/yum.repos.d/devtools-2.repo && \
-    yum install -y devtoolset-2-gcc devtoolset-2-binutils devtoolset-2-gcc-c++ && \
+    yum install -y centos-release-scl && \
+    yum install -y devtoolset-7 && \
+    yum install -y devtoolset-7-gcc devtoolset-7-binutils devtoolset-7-gcc-c++ && \
     # Python
     wget https://repo.continuum.io/miniconda/Miniconda2-4.3.27-Linux-x86_64.sh && \
     bash Miniconda2-4.3.27-Linux-x86_64.sh -b -p /opt/python && \
     # CMake
-    wget -nv -nc https://cmake.org/files/v3.6/cmake-3.6.0-Linux-x86_64.sh --no-check-certificate && \
-    bash cmake-3.6.0-Linux-x86_64.sh --skip-license --prefix=/usr
+    wget -nv -nc https://cmake.org/files/v3.13/cmake-3.13.0-Linux-x86_64.sh --no-check-certificate && \
```

Comment:
If possible lets use the latest cmake `3.17.2` everywhere or at least `3.14.0` (which is used in `container/Dockerfile.gpu`)